src/execution/DAG/dag.c

```diff
@@ -138,34 +132,12 @@ static int _DAG_PersistTensors(RedisModuleCtx *ctx, RedisAI_RunInfo *rinfo) {
     return REDISMODULE_OK;
 }
 
-static int _ModelSingleOp_PersistTensors(RedisModuleCtx *ctx, RAI_DagOp *op, RAI_Error *err) {
+static int _PersistTensors(RedisModuleCtx *ctx, RAI_DagOp *op, RAI_Error *err) {
```

Comment:
Maybe `_Dag_SingleOpPersistTensors` is better, as we do not call this in DAG commands contexts